source/localizable/_code-of-conduct.ar.md

```diff
@@ -0,0 +1,52 @@
+
+فإذا لاحظت شخصًا يبذل جهدًا حثيثًا لضمان أنَّ مجموعتنا مرحِّبةٌ أليفةٌ و يشجِّع الجميع على الانخراط بالحدِّ الأقصى، فأعلمونا بهم.
+
+السلوك المتوقّع
+------------
+* المشاركة بصدق و انخراط. فبذلك تساهمون في نمو و استمراية هذه المجموعة
+* الحساسيّة و تحرّي الاحترام في القول و الفعل
+* السعي إلى التعاون قبل الصراع
+* الترفُّع عن التحقير و التمييز و التحرّش بالقول و الفعل
+* مراعاة المحيط و المشاركين الآخرين.
+* تنبيه ميسّري المجموعة عند ملاحظة موقف خطر، أو شخص مأزوم، أو انتهاك مدوّنة السلوك هذه، حتّى و إن كانت تبدو عابرة.
+
+السلوك غير المقبول
+----------------
+السلوكيات غير المقبولة تشمل الترويع و التحرُّش و التسلُّط و التمييز و التحقير بالقول أو بالفعل مِنْ قِبَل أيِّ مشارك في مجموعاتنا سواء كانت المشارَكة سبرانية أو في الفعاليات و الملتقيات أو في التواصل الفردي الحاصل في مجال عمل المجموعة. أحيانا ما نتشارك محافِل فعالياتنا مع الجمهور العمومي، و عندها تجب مراعاة احترام كُلِّ رُوَاد المحفل.
+
+يدخل في نطاق التحرُّش: التعليقات الشفاهية أو المكتوبة المؤذية أو الحُكميّة فيما يتعلّق بالنوع الاجتماعي و الميل الجنسي و العرق و الدين و الإعاقة؛ و كذلك إساءة استخدام العُري أو الصور الجنسيّة (بما فيها في العروض لتقديمية) و مناظر العنف (بما فيها في العروض التقديمية) و الترويع و الترصُّد و الملاحقة، و التسجيل الاقتحامي بالفوتوغرافيا أو الصوت أو غيرهما، و الدأب على مقاطعة الجلسات أو غيرها مِنْ فعاليات، و التلامس الجسدي غير اللائق و الاهتمام الجنسي بغير قبول.
+
+تبِعات السلوك غير المقبول
+----------------------
+صدور سلوك غير مقبول مِنْ أيِّ عضو في المجموعة، بمَن فيهم الرُّعاة و مَنْ لهم سلطات اتّخاذ القرارات لَن يُتهاوَن معه. كُلُّ مَنْ يُطلَب منه الامتناع عن سلوك غير مقبول يجب عليه الانصياع فورًا.
+
+إذا انخرط عضو مِنْ المجموعة في سلوك غير مقبول فإنَّ المُيسِّرين قد يلجؤون إلى التصرُّفات التي يرونها مناسبة، بما في ذلك الحظر المؤقَّت أو الطرد الدائم مِنْ المجموعة بلا إنذار (و بلا إرجاع لقيمة أيَّ اشتراك مدفوع).
+
+إذا شهدت سلوكًا غير مقبول أو تعرَّضت له
+---------------------------------
+إذا تعرَّضت لسلوك غير مقبول أو كانت لديك مخاوف أُخرى فينبغي إخطار مُيسِّرٍ في أقرب فرصة. توجد قائمة أسماء الميسِّرين الواجب الاتصال بهم لكلِّ مِنْ المجموعات المتبنيّة مدوّنة السلوك هذه في ذيل الصفحة. علاوة على ذلك فإنَّ الميسرين متاحون لمساعدة أفراد المجموعة على مخاطبة جهات إنفاذ القانون أو تحقيق الاطمئنان لمَنْ يتعرَّضون لسلوك غير مقبول. في سياق الفعاليات ذات الحضور بالجسد فإنَّ المنظِّمين سيوفِّرون مرافقين لِمَنْ يتعرَّضون للكرْب.
+
+التظلُّم
+-----
+إذا شعرت بأنّك قد اتهمت زورًا أو ظُلمًا بمخالفة مدوّنة السلوك هذه فعليك مخاطبة أحد الميسِّرين بتظلّم مُحكَم، و سيُنظَر في تظلّمك بما يتوافق مع سياسات الحوكمة السارية للمجموعة.
+
+النطاق
+-----
+نتوقّع مِنْ كلِّ المشاركين (المساهمون، سواء دُفع لهم أجرُ أم لا، و الرُعاة، و الضيوف) الالتزام بمدونّة السلوك هذه في كلِّ محافل المجموعة -- السبرانية و الجسدية -- و كذلك في التواصل الفردي المتعلِّق بعمل المجموعة.
+
+الرخصة و الإشادات
+---------------
+مدونة برلين للسلوك منشورة برخصة المشاع الإبداعي النسبة-المشاركة بالمثل، و هي مبنية على [pdx.rb Code of Conduct](https://pdxruby.org/CONDUCT)
```

Comment:
I also noticed this url throws a 404 now

Reply:
The URL https://pdxruby.org/CONDUCT is working fine with me. Can you please check again?